modules/update-availability.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding-openshift-updates.adoc
+
+:_content-type: CONCEPT
+[id="update-availability_{context}"]
+= Common questions about update availability
+
+There are several factors that affect if and when an update is made available to an {product-title} cluster. The following list provides common questions regarding the availability of an update:
+
+*What are the differences between each of the different update channels?*
```

Comment:
Minor UX nit: `<h...>` headers vs. `<strong>`?  I'm not sure if there is an accessibility to a header's clearer semantics or not.  Also maybe add link anchors to these (which header tooling might help with?), so folks can easily get links when sharing pointers to specific answers, as long as there is unique text in the answers for these questions, instead of just links out for where to find the canonical text.

Reply:
the current implementation is meant for these lines to be bold and not explicit headers with anchor links, but I suppose that I can make that change now. I'll see what these questions look like as headers, but I'm not allowed to skip heading levels, so if the h3 font size is too large then I think I'll stick with keeping these bold

Reply:
I'm not above dropping into the browser-dev view to find an `id` anchor I can use in a URI fragment ;).  If you decide to stick with bold, adding link anchors would at least allow folks to link to specific questions, making life convenient for the link-receiving person, even if it was a bit more work for the link-generating person.

Reply:
I was going to save my update for my new PR but I'll put it here as well: I think I'll keep these questions as bold format, as my only option with headings would be to use the next heading level down, which would (imo) result in absurdly large font sizes for the questions. 

However, I can and will add anchor IDs above these questions for easier linking. Like you say, it's a bit more hassle for the person finding the anchor (and no telling if anyone will even know it exists, because who randomly peruses the HTML of a web page? haha), but I see the value of having it in general.